examples/sumo/bay_bridge.py

```diff
@@ -43,12 +43,16 @@ def bay_bridge_example(render=None,
     if render is not None:
         sumo_params.render = render
 
-    sumo_car_following_params = SumoCarFollowingParams(speedDev=0.2)
+    sumo_car_following_params = SumoCarFollowingParams(
+        speedDev=0.2,
+        speed_mode="all_checks",
+    )
     sumo_lc_params = SumoLaneChangeParams(
```

Comment:
I noticed that sometimes you perform this line of code in the "vechicle.add()" below - instead of performing this line separately as you do right here. Not sure if you wanted to be consistent with this or if it doesn't matter.